ansible/roles/kong-api/defaults/main.yml

```diff
@@ -126,7 +126,7 @@ kong_apis:
         - 'appAccess'
     - name: rate-limiting
       config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.hour: "{{ acceptTermsAndCondition_limit_per_hour if acceptTermsAndCondition_limit_per_hour else medium_rate_limit_per_hour }}"
```

Comment:
why can't we just have a `{{ var | d() }}` rather than the if else. So that it'll be idiomatic and easy to understand

Reply:
i think you saw an older commit @rjshrjndrn

Reply:
It is no more valid now